web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/propertyTax/Property/components/PendingAmountDue/index.js

```diff
@@ -31,28 +48,66 @@ class PendingAmountDialog extends Component {
   }
 
   render() {
-    const { open, closeDialogue, amount } = this.props;
+    const { open, closeDialogue, amount, waterDetails, sewerDetails, consumerCode } = this.props;
     const envURL = "/egov-common/pay";
     return (
       <Dialog
         open={open}
         children={[
-          <div style={{ margin: 16, display: "grid" }}>
-            <Label label="PT_PENDING_AMOUNT_DUE" fontSize="20px" labelClassName="pending-amount-due" />
+          <div style={{ margin: 16 }}>
+            <Label label="PT_PENDING_AMOUNT_DUE" style={labelStyle} labelClassName="pending-amount-due" />
             <br />
             <p className="dialog-content">
-              <Label fontSize="18px" className="dialog-content-label" label="PT_YOU_HAVE" />&nbsp;<Label className="dialog-content-label" labelClassName="dialog-content-label" fontSize="18px" label="PT_MUTATION_RS" />&nbsp;<div className="dialog-content-amount">{amount}</div>&nbsp;<Label className="dialog-content-label" label="PT_PENDING_AMOUNT" fontSize="18px" />
               <Label fontSize="18px" label="PT_INORDER_TO_TRANSFER" />
             </p>
-
-            <div className="text-right" style={{ marginTop: 10 }}>
-              <Label className="footer-amount" labelClassName="footer-amount" label="PT_MUTATION_RS" fontSize="24px" /><span className="footer-amount-no" > {amount}</span>&nbsp;&nbsp;&nbsp;&nbsp;
-              <Button
-                label={<Label buttonLabel={true} label="PT_PROCEED_TO_PAY" fontSize="16px" labelClassName="footer-button-label" />}
-                primary={true}
-                className="footer-button"
-                onClick={() => { this.navigateToCommonPay(envURL) }}
-              />
+            <div>
+              <Label fontSize="18px" label="PT_PROPERTY_DUE" />
+              <div style={payNowButton} >
+                <div style={labelStyle}>Rs {amount}</div>
```

Comment:
Use localisation key for Rs

similar to 
     <div className="text-right" style={{ marginTop: 10 }}>
              <Label className="footer-amount" labelClassName="footer-amount" label="PT_MUTATION_RS" fontSize="24px" /><span className="footer-amount-no" > {amount}</span>

Reply:
Added localization key for Rs